### PR TITLE
Header and footer sections

### PR DESCRIPTION
**Pull request by swengineermwi**
Created the header and footer sections of the Newsweek website as per Microverse directive. We have used Bootstrap and custom, CSS classes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            <div class="d-none d-lg-block col-lg-4">
+                <div class=" item-height d-flex align-items-center">
+                    <p class="text-light fs-16">Wed, Jan 20, 2021</p>
+                </div>
+            </div>
+
+            <div class="col-6 col-lg-4  item-height d-flex align-items-center justify-content-start justify-content-lg-center">
+                <div>
+                    <img src="logo.svg" alt="" class="logo">
+                </div>
```
Comment (Dipeshtwis) [inline index.html]: Newsweek logo should have a link.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            </div>
+
+            <div class="col-6 col-lg-4 item-height d-flex align-items-center justify-content-end">
+                <i class="fa fa-user text-light d-md-none"></i>
+
+                <div class="d-flex">
+                    <button type="button" class="btn d-none d-md-block  text-light fs-18"><strong>LOGIN</strong></button>
+                    <button type="button" class="btn btn-primary btn-sub text-dark"><strong>SUBSCRIBE<span class="heavy font-roboto">></span></strong></button>
```
Comment (Dipeshtwis) [inline index.html]: Login and Subscribe have links

File: index.html [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Experts</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Vantage</a>
+                    </li>
+                    <li>
+                        <div>
+                            <input type="text" id="search" name="search" class="search font-roboto" placeholder="Search">
+                            <label for="search"><i class="fa fa-search"></i></label>
```
Comment (Dipeshtwis) [inline index.html]: The search input must be wrapped in the form tag.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status: ♻️ Changes Required ♻️
## Hi @Mwila-Kaunda and @1v4n4, I'm Dipesh your reviewer for this PR.

Your clone is awesome. It looks like the exact one. I also have to appreciate you for your readme file and for fulfilling almost all the requirements in one shot. 👏 👏

There are some changes I'm suggesting here to make this project more awesome. see below.

### Always make a descriptive commit. This shows how and when you are implementing which section or feature.

## For Approval

## General

- [x] Instead of using 991px as second breakpoint. you should use 992px.
- [x] The element should have a proper gap in between them. For now, it is somehow looking ugly in all the views. Please make sure it looks like the same. 

Screenshot (1293)

## Header
- [x] I have mentioned in the comments.

## Footer

- [x] The Newsweek logo is showing only half. Please fix this issue. See in the above image.
- [x] There is one missing border below the Newsweek logo. please implement that.
- [x] Make social icons little bigger so that it looks like the same.

Fix this and when you are ready you can Request another code review. See you there.

## Reminder

**"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) now, you have only 2 code review left for this project."**

Code review by **Dipesh Kumar**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk)
[Twitter](https://twitter.com/97deepeshkumar)

File: index.html (hunk 1 above)
Comment (swengineermwi) [reply index.html]: Got it

File: index.html (hunk 3 above)
Comment (swengineermwi) [reply index.html]: Got it

File: index.html (hunk 2 above)
Comment (swengineermwi) [reply index.html]: got it

File: index.html [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        <li class="border-end-sm"><a href="#" class="text-secondary">Editorial Guidelines</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Advertise</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Copyright</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Terms & Conditions</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Privacy Policy</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Cookie Policy</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Terms of Sale</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Archive</a></li>
+        <li><a href="#" class="text-secondary">Announcements</a></li>
+      </ul>
```
Comment (Elbie-em) [inline index.html:178]: Hint: you can add bootstrap padding to each ul li items to closely match the original
**Original**
Screenshot 2021-04-06 at 11 00 45

**Yours**
Screenshot 2021-04-06 at 12 40 55

File: index.html [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+        </div>
+      </div>
+    </nav>
+  </header>
+
+  <footer class="c">
+    <div class="row border-bottom margin-y-md">
+      <div class="col-12 col-md-8 d-flex">
+        <div class="">
+          <img src="logo2.svg" class="footer-logo align-bottom" alt="">
```
Comment (Elbie-em) [inline index.html:117]: Kindly add a link to your news week logo

**Review by Elbie-em — APPROVED** [review]
## STATUS: APPROVED :white_check_mark:

Hi @Mwila-Kaunda and @1v4n4, congratulations on applying the changes requested.
I recommend you apply the optional changes below before you move on to you next milestone.

## [OPTIONAL]

- [ ] Kindly check my comments regarding your Newsweek logo in your footer
- [ ] Kindly check my comments regarding content-disposition changes in your footer.

## All the best with your next milestone :clap:

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

File: index.html (hunk 4 above)
Comment (swengineermwi) [reply index.html:178]: Alright, we are on it!

**Pull request merged**